src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -177,9 +175,10 @@
                 </DataGridTemplateColumn>
             </DataGrid.Columns>
         </DataGrid>
-        <GridSplitter Grid.Row="2" HorizontalAlignment="Stretch"></GridSplitter>
+        <GridSplitter Grid.Row="3" HorizontalAlignment="Stretch" Margin="0,28,0,7"/>
```

Comment:
why this change?

Reply:
note: I think this looks like an accidental change and should probably be reverted before merging. It looks like this would break the grid splitter.

@allisonchou Is it possible you used the designer and changed this accidentally? Or is it an intentional change? Thanks